examples/tfengine/full.yaml

```diff
@@ -18,6 +18,10 @@ data:
   ORG_ID: "123"
   BILLING_ACCOUNT: "000-000-000"
   STORAGE_LOCATION: "us-central1"
+  CLUSTER_REGION: "us-central1"
```

Comment:
Can this be overriden per-cluster? It would be useful to not force all clusters to be in the same region.

Reply:
Will do in follow up PR based on Xin's comments below.